src/components/CommentCard.jsx

```diff
@@ -1,9 +1,36 @@
+import { useContext, useState } from "react";
+import { UsernameContext } from "../contexts/Username";
+import { deleteUsersComment } from "../api";
+
 function CommentCard({ comment }) {
+	const { username } = useContext(UsernameContext);
+	const [optimisticDeleted, setOptimisticDeleted] = useState(false);
+	const comment_id = comment.comment_id;
+
+	function handleDeleteClick() {
+		setOptimisticDeleted(true);
```

Comment:
Its fine to optimistically say deleting... but not to say deleted as things may fail and the change wont persist. 

Instead, on click, I would say "deleting..." then in the .then() block you can say "message has been deleted"